yurllib/aasa.go

```diff
@@ -185,23 +215,27 @@ func evaluateAASA(result []byte, contentType []string, bundleIdentifier string,
 
 	err := json.Unmarshal(result, &reqResp)
 	if err != nil {
-		if len(contentType) > 0 && contentType[0] == "application/pkcs7-mime" {
-			_, err := pkcs7.Parse(result)
-			if err != nil {
-				formatErrors = append(formatErrors, fmt.Errorf("PKCS7 Parse Fail: \n%w", err)) //define this better
-				return output, formatErrors
-			}
-		} else {
-			prettyJSON, err := json.MarshalIndent(result, "", "    ")
-			if err != nil {
-				formatErrors = append(formatErrors, fmt.Errorf("io.ReadAll failed to parse with error: \n%w", err)) //define this better
+		if len(contentType) > 0 {
+			if contentType[0] == "application/pkcs7-mim" {
```

Comment:
@matthew-meyer-au Obviously it is not intentional 🙂 Thanks for picking that up. Fixed here https://github.com/BlinqOSS/yurl/pull/1/commits/81f981c026d68e6ac0381db7ca03c2be48d41c15.

I thought of adding some unit tests for `pkcs7` mime type but looks like this is only for _super old_ AASA file. I could only find references to it in some forum threads from 2018 and before. I decided it isn't worth the effort to do so, since we might remove support for it soon.